README.md

```diff
@@ -91,17 +90,43 @@ Options:
   -V, [--verbose], [--no-verbose]    # Verbose output for debugging purposes
 
 initializes folder structure
-```
+~~~
 <!-- END_HELP_COMMAND_INIT -->
 
-### Generate RBI files for gems
+## Usage
+
+### Generating RBI files for gems
+
+Sorbet does not read the code in your gem dependencies, so it does not know the constants and methods declared inside gems. Tapioca is able to load your gem dependencies from you application `Gemfile` and compile RBI files to represent their content.
 
-Command: `tapioca gem [gems...]`
+To generate the RBI files for the gems used in your application run the following command:
 
-This will generate RBIs for the specified gems and place them in the RBI directory.
+~~~sh
+$ bin/tapioca gems [gems...]
+
+Removing RBI files of gems that have been removed:
+
+  Nothing to do.
+
+Generating RBI files of gems that are added or updated:
+
+  Requiring all gems to prepare for compiling...    Done
+
+  Compiled ansi
+      create  sorbet/rbi/gems/ansi@1.5.0.rbi
+
+  ...
+
+All operations performed in working directory.
+Please review changes and commit them.
+~~~
+
+This will load your application, find all the gems required by it and generate an RBI file for each gem under the sorbet/rbi/gems directory. This will also import the signatures that can be found inside each gem source code as well as the YARD documentation of the gem.
```

Comment:
Have we turned on YARD documentation by default? If not, should we mention that it requires turning it on?

Reply:
Good call: https://github.com/Shopify/tapioca/pull/920